src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd

```diff
@@ -50,104 +51,99 @@ if ERRORLEVEL 1 (
-echo Grant app pool %PRServiceName% "Modify" access to %CurrentRepo% and its subdirectories
-call :Run icacls %CurrentRepo% /grant:r "IIS APPPOOL\%PRServiceName%":M /T /Q
+:: Grant app pool %_prServiceName% "Modify" access to %_currentRepo% and its subdirectories
+echo Grant app pool %_prServiceName% "Read and Execute" access to the PRService and its subdirectories
+call :Run icacls %_currentRepo%\src\System.Private.ServiceModel\tools\PRService /grant:r "IIS APPPOOL\%_prServiceName%":(OI)(CI)RX /Q
 if ERRORLEVEL 1 goto :Failure
-:: Uncomment below and delete above when we switch to central PR service
-REM echo Grant app pool %PRServiceName% "Read and Execute" access to the PRService and its subdirectories
-REM call :Run icacls %CurrentRepo%\src\System.Private.ServiceModel\tools\PRService /grant:r "IIS APPPOOL\%PRServiceName%":RX /T /Q
-REM if ERRORLEVEL 1 goto :Failure
 
 :SkipPRService
 
 :: Grant master PR service's app pool access to current repo if master PRService exists
-if /I '%MasterRepo%'=='%CurrentRepo%' goto :SkipGrantAccess
-if NOT EXIST %MasterRepo% goto :SkipGrantAccess
-%appcmd% list apppool %PRServiceMaster% >nul
+if /I '%_masterRepo%'=='%_currentRepo%' goto :SkipGrantAccess
+if NOT EXIST %_masterRepo% goto :SkipGrantAccess
+%_appcmd% list apppool %_prServiceMaster% >nul
 if ERRORLEVEL 1 goto :SkipGrantAccess
-echo Grant master PRServer app pool: %PRServiceMaster% "Modify" access to %CurrentRepo% and its sudirectories
-call :Run icacls %CurrentRepo% /grant:r "IIS APPPOOL\%PRServiceMaster%":M /T /Q
+echo Grant master PRService app pool: %_prServiceMaster% "Modify" access to %_currentRepo% and its sudirectories
+call :Run icacls %_currentRepo% /grant:r "IIS APPPOOL\%_prServiceMaster%":(OI)(CI)M /Q
 if ERRORLEVEL 1 goto :Failure
 
 :SkipGrantAccess
 
-:: Install certificates if %WcfTestDir% does not exist; otherwise, assume certs are installed already
-if EXIST %WcfTestDir% goto :SkipCertInstall
+:: Install certificates if %_wcfTestDir% does not exist; otherwise, assume certs are installed already
+if EXIST %_wcfTestDir% goto :SkipCertInstall
 
 :: Make sure there is an HTTPS binding in IIS site "Default Web Site"
-%appcmd% list site "Default Web Site" /text:* | findstr /i bindings | findstr /i https >nul
+%_appcmd% list site "Default Web Site" /text:* | findstr /i bindings | findstr /i https >nul
 if ERRORLEVEL 1 (
     echo Add an HTTPS binding to "Default Web Site"
-    call :Run %appcmd% set site "Default Web Site" /+bindings.[protocol='https',bindingInformation='*:443:']
+    call :Run %_appcmd% set site "Default Web Site" /+bindings.[protocol='https',bindingInformation='*:443:']
     if ERRORLEVEL 1 goto :Failure
 )
 
-:: Use current repo for cert gen if master repo does not exist
-if NOT EXIST %MasterRepo% (
-    set MasterRepo=%CurrentRepo%
-    set WcfServiceMaster=%WcfServiceName%
+:: Use current repo for cert generation if master repo does not exist
+if NOT EXIST %_masterRepo% (
+    set _masterRepo=%_currentRepo%
```

Comment:
I know this is the same logic as last time, but it seems slightly dangerous to set _masterRepo when it isn't really the master repo.  There are other checks that %_masterRepo% != %_currentRepo%, so the pattern has been established they not be the same elsewhere.  Not blocking but would prefer creating a new env var like %_setupRepo% (or something like that) for the BuildCertUtil.cmd and CertificateGenerator.exe below.